prisma/schema.prisma

```diff
@@ -7,11 +7,62 @@ generator client {
   provider = "prisma-client-js"
 }
 
-model User {
-  username String @id
-  name     String
-  email    String @unique
+model Users {
+  username  String @id
+  firstName String
+  lastName  String
+  email     String @unique
 
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  createdAt    DateTime       @default(now())
+  updatedAt    DateTime       @updatedAt
+  subdomians   Subdomains[]
+  certificates Certificates?
+}
+
+model Subdomains {
+  id          Int             @id @default(autoincrement())
+  username    String
+  name        String
+  type        SubdomainType
+  value       String
+  description String
+  course      String
+  ports       String
+  createdAt   DateTime        @default(now())
+  updatedAt   DateTime        @updatedAt
+  expiresAt   DateTime
+  status      SubdomainStatus
+  user        Users           @relation(fields: [username], references: [username])
+}
+
+model Certificates {
+  id          Int          @id @default(autoincrement())
+  username    String       @unique
+  subject     String
+  certificate String
+  privkey     String
+  validFrom   DateTime     @default(now())
+  validTo     DateTime 
+  user        Users        @relation(fields: [username], references: [username])
+  challange   Challenges[]
+}
+
+model Challenges {
```

Comment:
I'll get @dadolhay to weigh in on this one, I'm not sure what the conversation was about this.

Reply:
We discussed this with @cychu42, and with the lack of a record ID from route53 this is currently the best we could come up with. If later other requirements come in, we can alter the schema, but I think for now this is good.

Reply:
Sounds good